CoreJavaProject/build/classes/DBProperties

```diff
@@ -0,0 +1,4 @@
+DBDriver=oracle.jdbc.driver.OracleDriver
+DBName=jdbc:oracle:thin:@localhost:1522:xe
+User=scott

```

Comment:
You might consider using Volt or other secret holders(AWS SecretManager for example).

Reply:
Yes definitely